cpp/src/io/parquet/writer_impl.cu

```diff
@@ -82,6 +83,17 @@ parquet::Compression to_parquet_compression(compression_type compression)
   }
 }
 
+template <typename T>
+void print(rmm::device_uvector<T> const& d_vec, std::string label = "")
+{
+  std::vector<T> h_vec(d_vec.size());
+  cudaMemcpy(h_vec.data(), d_vec.data(), d_vec.size() * sizeof(T), cudaMemcpyDeviceToHost);
+  printf("%s (%lu)\t", label.c_str(), h_vec.size());
+  for (auto&& i : h_vec)
+    std::cout << i << " ";
+  printf("\n");
+}
+
```

Comment:
```suggestion
```

Reply:
I remember removing this. How did it get back in here?